src/universal/components/LoadingView/LoadingView.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import look, {StyleSheet} from 'react-look';
+import theme from 'universal/styles/theme';
+import Pato from 'universal/styles/theme/images/graphics/pato.svg';
+
+let styles = {};
+
+const patoHop = StyleSheet.keyframes({
+  '0%': {
+    transform: 'translate(0, 0)'
```

Comment:
back in the day i used to use `translate3d` to force hardware acceleration (draws the animation in its own composite layer). Nowadays, I hear `will-change` is all the range, but isn't supported by Microsoft stuff. @ackernaut what's your thought? I wanna make sure those duckies are jumping at a smooth 60fps :wink: